internal/authn/authn.go

```diff
@@ -3,7 +3,7 @@ package authn
 import (
 	"context"
 
-	"github.com/openfga/openfga/pkg/authclaims"
+	"github.com/openfga/openfga/pkg/authcontext"
```

Comment:
This PR has gotten too large, would it be possible to move the renames that are not critical to this PR to a separate PR that can go directly to `main`? 🙏

Reply:
Will do

Reply:
Removed